web-components/src/components/section/index.tsx

```diff
@@ -31,32 +33,36 @@ interface StyleProps {
   titleAlign?: 'left' | 'right' | 'inherit' | 'center' | 'justify' | undefined;
 }
 
+const Root = styled(Box, {
+  name: 'RegenSection',
+  shouldForwardProp: prop => prop !== 'sx' && prop !== 'withSlider',
```

Comment:
because there's no `useStyles` to pass props to, they need to be passed to the component directly. `shouldForwardProps` is needed to not pass them to underlaying component